src/modules/oaf/components/OafFilter.js

```diff
@@ -55,21 +58,23 @@ export class OafFilter extends MvuElement {
 				return { ...model, minValue: data };
 			case Update_Max_Value:
 				return { ...model, maxValue: data };
+			case Update_Expression:
+				return { ...model, expression: createOafFilterExpression(model) };
 		}
 	}
 
 	onInitialize() {
-		this.observeModel('operator', () => this.dispatchEvent(new CustomEvent('change')));
-		this.observeModel('minValue', () => this.dispatchEvent(new CustomEvent('change')));
-		this.observeModel('maxValue', () => this.dispatchEvent(new CustomEvent('change')));
-		this.observeModel('value', () => this.dispatchEvent(new CustomEvent('change')));
+		this.observeModel('operator', () => this._onModelChange());
+		this.observeModel('minValue', () => this._onModelChange());
+		this.observeModel('maxValue', () => this._onModelChange());
+		this.observeModel('value', () => this._onModelChange());
```

Comment:
1. As discussed, let's check if we can avoid the double-rendering

2.`this.observeMode` also takes an array as an argument